EventsExpress.Test/ServiceTests/UnitOfMeasuringServiceTests.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using EventsExpress.Core.DTOs;
+using EventsExpress.Core.Exceptions;
+using EventsExpress.Core.Services;
+using EventsExpress.Db.Entities;
+using EventsExpress.Test.ServiceTests.TestClasses.UnitOfMeasuring;
+using Moq;
+using NUnit.Framework;
+using NUnit.Framework.Constraints;
+
+namespace EventsExpress.Test.ServiceTests
+{
+    internal class UnitOfMeasuringServiceTests : TestInitializer
+    {
+        private UnitOfMeasuringService service;
+        private UnitOfMeasuringDTO correctUnitOfMeasuringDTO;
+        public string createUnitName = "create Unit Name";
```

Comment:
Public fields must start with capital letter and can be deifned as `const`. Please do that.